lms/templates/student_account/login.underscore

```diff
@@ -1,14 +1,24 @@
 <div class="js-form-feedback" aria-live="assertive" tabindex="-1">
 </div>
 
-<% if ( context.createAccountOption !== false && !context.syncLearnerProfileData) { %>
+<% if ( context.createAccountOption !== false && !context.syncLearnerProfileData && context.enterpriseName == "") { %>
```

Comment:
Why not `!context.enterpriseName` just in case enterpriseName somehow gets in as undefined or null and also to match the check for syncLearnerProfileData